Makefile

```diff
@@ -100,6 +100,7 @@ test-in-docker:
 		-v `readlink -f $(resultsdir)`:/test-run-results:z \
 		-e NAMESPACE \
 		`env | grep _3SCALE_TESTS_ | sed 's/^/-e /'` \
```

Comment:
This can be `...| grep -e _3SCALE_TESTS_ -e ^flags= |...`

Reply:
or not